plugins/cache.rst

```diff
@@ -103,7 +105,22 @@ removed from the cache::
         'cache_lifetime' => 86400*365, // one year
     ];
 
+Caching of different request methods
+````````````````````````````````````
 
+Most of the time you should not change the ``methods`` option. However if you are working for example with HTTPlug
+based SOAP client you might want to additionally enable caching of POST requests::
+
+    $options = [
+        'methods' => ['GET', 'HEAD', 'POST'],
+    ];
+
+You can cache any valid request method.
```

Comment:
maybe:

"methods" overwrites the default methods, if you want to keep caching GET and HEAD requests, you need to repeat them. You can specify any request method that conforms to RFC-7230. Note that request methods are case sensitive, so `post` and `POST` are not the same.